src/sentry/models/projectredirect.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import Model, FlexibleForeignKey
+
+
+# TODO: Should this be ProjectSlugHistory
+class ProjectRedirect(Model):
+    __core__ = True
+
+    redirect_slug = models.SlugField(null=True, db_index=True)
```

Comment:
why can this be null?

Reply:
734a6d7 You're right it should not be null.